package.json

```diff
@@ -20,7 +20,12 @@
     "analyze": "./script/webpack-analyze"
   },
   "dependencies": {
+<<<<<<< HEAD
     "@theforeman/vendor": "^12.0.1",
+=======
+    "@module-federation/utilities": "^1.7.0",
+    "@theforeman/vendor": "^13.0.1",
+>>>>>>> 7008091a7 (Fixes #37312 - Upgrade foreman-vendor packages to 13.x)
```

Comment:
this doesn't look correct, and we don't need module federation utilities in 3.9

Reply:
That was def my fault, I should have caught the `=====` and the `>>>` fixed up